### PR TITLE
Feature/mobile app page

### PR DESCRIPTION
**Pull request by Shemsedin**
### Summary
This is about creating of Create mobile app page (issue#6) as per this [link](https://github.com/quran/quran.com-frontend-next/issues/6) 

### Screenshots

| Before | After |
| ------ | ------ |
| [image here] | [image here] |

### CODE REVIEW

**Comment by tifa2UP** [comment]
Assalamu alaikum Shamsedin, JAK for creating the PR. it looks like the build is failing. Design is working on an overhaul for the design system and main pages (including the mobile apps). The goal is to have quran-next designed for an overall better experience and not just be a re-skin over the existing website. 

Would you mind if I keep the PR open until we get the specs from Design? This way it'd be easier to refactor them one at a time. Will update the Github issues accordingly

File: .idea/$CACHE_FILE$ [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (abdullahceylan) [inline .idea/$CACHE_FILE$:1]: I think you shouldn't send `.idea` folder as it's for your IDE. So can you add it to the `.gitignore`?

File: src/components/Pages/QuranApps/AppsPage.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```
Comment (abdullahceylan) [inline src/components/Pages/QuranApps/AppsPage.stories.tsx:1]: Within React 17 and newer versions come with the new JSX transforms and you don't need to import `React` anymore.

You can update the other files in this PR as well.

File: .idea/$CACHE_FILE$ (hunk 1 above)
Comment (Shemsedin) [reply .idea/$CACHE_FILE$:1]: @abdullahceylan  I have added this to gitignore

File: src/components/Pages/QuranApps/AppsPage.stories.tsx (hunk 2 above)
Comment (Shemsedin) [reply src/components/Pages/QuranApps/AppsPage.stories.tsx:1]: I tried to remove this but it `yarn lint:` through an error and then doesn't let me commit hence can't remove the import.

Comment (abdullahceylan) [reply src/components/Pages/QuranApps/AppsPage.stories.tsx:1]: Salaam @Shemsedin,

If you add `react/jsx-uses-react: 0` right after line 47 in `.eslintrc.yml` you won't get an error anymore.

Comment (Shemsedin) [reply src/components/Pages/QuranApps/AppsPage.stories.tsx:1]: I tried this but it is the same. Maybe if you tray on your end what works (on this branch) and let me know and I am happy to change that part.

**Pull request closed without merging**